pkg/cli/cli.go

```diff
@@ -119,7 +119,17 @@ type ConnQr struct {
 // set up. Logging is allowed.
 func (f *Flags) Post(cfg *config.Instance, pl platforms.Platform) {
 	if *f.Config {
+		client.LocalClient(
+			cfg,
+			models.MethodSettingsUpdate,
+			"{\"runZapScript\":false}",
+		)
```

Comment:
when entering the UI disable the ability to run cards, so we don't mess up. We restore it after we are done with the UI.
Local client talk to the running version of zaparoo, while the UI spans an instance of zaparoo without the service active.